server/chat-commands/info.ts

```diff
@@ -2396,7 +2496,7 @@ const commands = {
 		// XXX: target is trimmed by Chat#splitMessage. Let's not add another
 		// awful hack like ! or help command keys for whether or not the target
 		// is raw for now.
-		target = this.message.substr(this.cmdToken.length + this.cmd.length + +this.message.includes(' ')).trimEnd();
+		target = this.message.substr(this.cmdToken.length + this.cmd.length + +this.message.includes(' ')).trimRight();
```

Comment:
Why was this changed? According to MDN, the standard name is `trimEnd`.

Reply:
according to VS code with an up-to-date package, trimEnd doesnt exist